pkg/command/query_manager.go

```diff
@@ -0,0 +1,20 @@
+package command
+
+import "fmt"
+
+type QueryManager map[string]*Query
+
+func (q QueryManager) Get(file string) *Query {
```

Comment:
Since we have a new _QueryManager_ to _get_ the queries, what if the logic of _creating_ the queries were moved as part of the manager as well?
Eg;
```go
 func (qm QueryManager) Add(file string) error {
	rawQuery, err := ioutil.ReadFile(filePath)
	if err != nil {
		return err
	}

        var query Query
	if strings.HasSuffix(filePath, ".sql") {
		query = Query{
			Raw:  string(rawQuery),
			Type: SQL,
		}
	}

	if strings.HasSuffix(filePath, ".promql") {
                query = Query{
			Raw:  string(rawQuery),
			Type: Prometheus,
		}
	}

        qm[strings.TrimLeft(strings.ReplaceAll(path, queriesDirAbs, ""), "/")] = &query

	return nil
}
```